scripts/provisioning/configcmd.py

```diff
@@ -99,6 +101,42 @@ def process(self, configure_only_openldap = False, configure_only_haproxy = Fals
     except Exception as e:
       raise S3PROVError(f'process() failed with exception: {e}')
 
+  def configure_s3_schema(self):
+    self.logger.info('openldap s3 configuration started')
+    cmd = ['/opt/seagate/cortx/s3/install/ldap/s3_setup_ldap.sh',
+           '--ldapadminpasswd',
+           f'{self.ldap_passwd}',
+           '--rootdnpasswd',
+           f'{self.rootdn_passwd}',
+           '--skipssl']
+    handler = SimpleProcess(cmd)
+    stdout, stderr, retcode = handler.run()
+    self.logger.info(f'output of setup_ldap.sh: {stdout}')
+    if retcode != 0:
+      self.logger.error(f'error of setup_ldap.sh: {stderr}')
+      raise S3PROVError(f"{cmd} failed with err: {stderr}, out: {stdout}, ret: {retcode}")
+    else:
+      self.logger.warning(f'warning of setup_ldap.sh: {stderr}')
+
+    if os.path.isfile("/opt/seagate/cortx/s3/install/ldap/rsyslog.d/slapdlog.conf"):
```

Comment:
Openldap logging configuration need to be moved to openldap mini provisioning

Reply:
it is present in openldap mini prov. I forgot to remove it from this place